geometry_msgs/Pose boxPose
geometry_msgs/Pose goalPose
---
bool output
